samples/net/sockets/http_server/prj.conf

```diff
@@ -78,3 +78,10 @@ CONFIG_NET_TCP_TIME_WAIT_DELAY=0
 # Device drivers
 CONFIG_GPIO=y
 CONFIG_LED=y
+
+# Network debug config
+CONFIG_LOG_MODE_IMMEDIATE=y
+CONFIG_NET_SOCKETS_LOG_LEVEL_DBG=n
+CONFIG_NET_HTTP_LOG_LEVEL_DBG=y
```

Comment:
I suggest not enable this by default, user can easily enable various debugging prints when compiling the application.

Reply:
Fixed.